### ISSUE TITLE
OMB Info - Sketch artifacts

### ISSUE THREAD

**Issue by caw310** [department-of-veterans-affairs/vets-design-system-documentation#913]
## Configuring this issue
- [ ] Add issue to appropriate epic
- [ ] Add Design System component label (such as `va-alert`)
- [ ] Add `component-new` or `component-update` label, if applicable
- [ ] Complete sections below
- [ ] Delete this section once complete

## Description
During the consistency audit, it was discovered that OMB info is not in the Sketch library.

## Details
Add OBM Info to the Sketch library

## Tasks
- [x] Create or update Sketch artifacts for component
- [x] Add Sketch artifacts to Sketch library, if not already there
- [x] Create component page in Sketch, if necessary
- [x] Add link to Sketch component page to documentation ticket

## Acceptance Criteria
- [x] Sketch component artifacts have been created or updated
- [x] Sketch component page has been created, if necessary
- [x] Link to Sketch component page has been added to component documentation ticket

**Comment by humancompanion-usds** [comment]
The symbol doesn't quite work the way I expected. When I overrode the Respondent burden I got this result:

![Screen Shot 2022-06-29 at 9.03.27 AM.png](https://images.zenhubusercontent.com/623c98bb27734da1243c0bb5/8ba5494b-007e-4052-b19d-93da41a21897)

I would expect just the values, not the labels, to be possible to override. I know it is annoying and I'm being pedantic but even though it's just a text example the symbol should be built a bit more defensively so it is harder for folks to accidentally mess it up.

**Comment by GnatalieH** [comment]
Hi @humancompanion-usds. I think I got the new OMB symbols to work as you'd requested! I was able to remove overrides for the labels and Privacy Act Statement, so just the values are editable (great idea). Because I had to remove overrides for the labels, I can't make just one symbol where the Designer can delete the respondent burden or OMB number depending on their use case. So, I made three symbols.
